frontend/sozisel-app/generator_utils/index.ts

```diff
@@ -0,0 +1,338 @@
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+const fs = require("fs");
+
+function getArgs() {
+  const args = {};
+  process.argv.slice(2, process.argv.length).forEach((arg) => {
+    if (arg.slice(0, 2) === "--") {
+      const longArg = arg.split("=");
+      const longArgFlag = longArg[0].slice(2, longArg[0].length);
+      const longArgValue = longArg.length > 1 ? longArg[1] : true;
+      args[longArgFlag] = longArgValue;
+    }
+  });
+  return args;
+}
+
+const args = getArgs();
+const moduleName = args["moduleName"];
+if (!moduleName) {
+  throw new Error(
+    "--moduleName argument is required. Example: --moduleName=Foo"
+  );
+}
+const displayName = args["displayName"];
+if (!displayName) {
+  throw new Error(
+    "--displayName argument is required. Example --displayName=Bar"
+  );
+}
+
+
+/*
+ * Updates hooks
+ */
+const locales = fs
+  .readFileSync("./public/locales/pl/common.json")
+  .toString()
+  .replace(
+    `"EventType": {`,
+    `"EventType": {"${moduleName}": "${displayName}",`
+  );
+fs.writeFileSync("./public/locales/pl/common.json", locales);
+const useGetEventTypeMessage = fs
+  .readFileSync("./src/hooks/useGetEventTypeMessage.ts")
+  .toString()
+  .replace(
+    "// placeholder for new module",
+    `case EventType.${moduleName}:
+return t("components.EventType.${moduleName}");\n// placeholder for new module`
+  );
+fs.writeFileSync(
+  "./src/hooks/useGetEventTypeMessage.ts",
+  useGetEventTypeMessage
+);
+const useGetEventTypename = fs
```

Comment:
Can you make a helper function that takes a src filename, destination filename, phrase to search for and to replace it with? You would not have to replace yourself so many times in this file otherwise